app/services/insights.js

```diff
@@ -1,8 +1,8 @@
 const appInsights = require('applicationinsights')
 
 function setup () {
-  if (process.env.APPINSIGHTS_CONNECTIONSTRING) {
-    appInsights.setup(process.env.APPINSIGHTS_CONNECTIONSTRING).start()
+  if (process.env.APPINSIGHTS_INSTRUMENTATIONKEY) {
+    appInsights.setup().start()
```

Comment:
The instrumentation key was deprecated and was due to be removed March 2025.  I haven't confirmed that it's stopped working, but recommend leaving this as connection string as that's the only official supported way.